components/membership/certificates-client-impl/src/main/kotlin/net/corda/membership/certificate/client/impl/HostedIdentityEntryFactory.kt

```diff
@@ -64,12 +66,12 @@ internal class HostedIdentityEntryFactory(
     }
 
     private fun getCertificates(
-        actualTlsTenantId: String,
+        certificateHoldingId: ShortHash?,
         certificateChainAlias: String,
     ): List<String> {
-        val certificateChain = retrieveCertificates(actualTlsTenantId, certificateChainAlias)
+        val certificateChain = retrieveCertificates(certificateHoldingId, CertificateUsage.P2P_TLS, certificateChainAlias)
             ?: throw CertificatesResourceNotFoundException(
-                "Please import certificate chain into $actualTlsTenantId with alias $certificateChainAlias"
+                "Please import certificate chain into p2p-tls with alias $certificateChainAlias"
```

Comment:
```suggestion
                "Please import certificate chain with usage "p2p-tls" and alias $certificateChainAlias"
```
just a suggestion to make it a bit more clear to the user what `p2p-tls` refers to.